packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -420,6 +422,9 @@ export function _addResourceSpans(
   if ('renderBlockingStatus' in entry) {
     data['resource.render_blocking_status'] = entry.renderBlockingStatus;
   }
+  if (resourceOrigin) {
+    data['span.domain'] = resourceOrigin;
```

Comment:
@AbhiPrasad I'm thinking about how we should domain this. Currently no convention for this exists here 
https://develop.sentry.dev/sdk/performance/span-data-conventions/
Although we should probably prefix this with `http.` instead of the internal `span.domain` we use.

If we wanted to follow open telementry spec https://opentelemetry.io/docs/concepts/signals/traces/
We could have `http.host` and `http.scheme`?

Reply:
You're right that we should not set `span.domain`.

The OTEL spec has diverged. Looking at https://opentelemetry.io/docs/specs/semconv/url/url/ and https://opentelemetry.io/docs/specs/semconv/http/http-spans/ we probably want to use `url.scheme` and `server.address` (`url.origin` feels better to me, but better to match OTEL).

We also should attach extra metadata that indicates that it is same origin, perhaps `url.same_origin`? This is probably super useful for processing, and allows us to easily tell what is 1st party vs. 3rd party resources without needing user involvement.